src/GEOUNED/Write/Functions.py

```diff
@@ -162,7 +186,8 @@ def writeSequenceOMCPY(Seq,prefix='S'):
      return line
 
 def MCNPSurface(id,Type,surf):
-    MCNP_def = ''
+    Serpent_def = ''
```

Comment:
This change is wrong. Should be 
MCNP_def = ''

Reply:
good spot!